pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -0,0 +1,48 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type OperatorSourceList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata"`
+	Items           []OperatorSource `json:"items"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type OperatorSource struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+	Spec              OperatorSourceSpec   `json:"spec"`
+	Status            OperatorSourceStatus `json:"status,omitempty"`
+}
+
+type OperatorSourceSpec struct {
+	// Type of operator source.
+	Type string `json:"type,omitempty"`
+
+	// Endpoint points to the remote app registry server from where operator manifests can be fetched.
+	Endpoint string `json:"endpoint,omitempty"`
+
+	// RegistryNamespace refers to the namespace in app registry. Only operator manifests under
+	// this namespace will be vsible. This is not a k8s namespace.
```

Comment:
`visible`.
I would say `Please note that this is not a k8s namespace.`

Reply:
Fixed